### PR TITLE
adding grafana dashboards for self-hosted event gateway

### LINKED ISSUE #427: Add example Grafana dashboard JSON
EG exposes some [Prometheus metrics](https://github.com/serverless/event-gateway/blob/master/docs/prometheus-metrics.md). For users that are running the Event Gateway themselves, we should have an exported [Grafana Dashboard](http://docs.grafana.org/reference/dashboard/) file to allow for quick monitoring setup.

### PR DESCRIPTION
**Pull request by sebito91**
## What did you implement:

Closes #427 

Adds the following grafana dashboards for `promeheus` and `influxdb`:
1. self-hosted event gateway (all spaces)
2. self-hosted event gateway (by space)

## How did you implement it:
Adding in dashboard templates that show stats exposed by the event gateway. These metrics are displayed in promeheus format but providing both `prometheus` and `influxdb` dashboard for the same data.

## How can we verify it:

Follow the `grafana` upload/import [instructions](http://docs.grafana.org/reference/export_import/#importing-a-dashboard) to test out the dashboards. **NOTE:** expects the default datasource.

## Todos:

- [x] Write tests
- [ ] Write documentation
- [x] Fix linting errors
- [x] Make sure code coverage hasn't dropped
- [x] Provide verification commands / resources
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

File: contrib/helm/event-gateway/values.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 image:
   repository: serverless/event-gateway
-  tag: 0.9.0
+  tag: latest
```
Comment (mthenw) [inline contrib/helm/event-gateway/values.yaml]: I think we should revert this change

Comment (sebito91) [reply contrib/helm/event-gateway/values.yaml]: Why? latest is set to v0.9.1...you don't want people deploying latest EG?

Comment (mthenw) [reply contrib/helm/event-gateway/values.yaml]: Nevermind. It's fine :D
